twitterclone.css

```diff
@@ -0,0 +1,208 @@
+body {
```

Comment:
Overall good work. There are few feedbacks with the page:
1. Good use of different font that is closer to what Twitter page uses. 
2. Good centering of elements on both sides 
3. That bird background is applied to blue side, but if my browser is in full screen, it is repeated twice.
4. Try to use icon tag on next projects to create icons next to "explore", "bookmarks", etc. 
5. Page does not respond well when browser is resized which can be mitigated with use of flex. 
Other than above feedbacks, page looks fine and html elements are well organized with divs. Keep trying new CSS styles and you will create cool looking pages in no time.